BookingsApi/BookingsApi/Controllers/V1/HearingsController.cs

```diff
@@ -209,6 +209,19 @@ public async Task<IActionResult> UpdateHearingsInGroup(Guid groupId, [FromBody]
                 
                 await _updateHearingService.UpdateEndpointsV1(requestHearing.Endpoints, hearing);
             }
+            
+            var hearings = request.Hearings.ToList();
+            var totalDays = hearings.Count;
+            var nextFirstHearingId = hearings[0].HearingId;
```

Comment:
`nextFirstHearing` doesn't make much sense, can we call it `firstDayHearing` instead

Reply:
no we can't call it like that. It is the next first day hearing and not the first day hearing. This is because is when the update is from the 2 /3/4/... day onward

Reply:
changed